### PR TITLE
bumping to 3.2 rc1

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

**Review by jlsherrill — APPROVED** [review]
ACK

File: repos/katello-repos.spec [hunk 1]
```diff
@@ -17,6 +17,9 @@ BuildArch:      noarch
 BuildRequires: sed
 BuildRequires: ruby
 BuildRequires: python
+%if 0%{?fedora} > 23
+BuildRequires: rubygems
+%endif
```
Comment (ehelms) [inline repos/katello-repos.spec:22]: Why is this required for the katello-repos RPM?

File: katello/katello.spec [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 Name:       katello
 Version:    3.2.0
-Release:    2.nightly%{?dist}
+Release:    2.rc1%{?dist}
```
Comment (ehelms) [inline katello/katello.spec:9]: Should probably bump the release number here to be safe.

**Review by ehelms — COMMENTED** [review]
I know this got merged already, but worth considering for updates.

File: repos/katello-repos.spec (hunk 1 above)
Comment (johnpmitsch) [reply repos/katello-repos.spec:22]: fedora 24 added a rubypick package (ruby version manager) that is required to be installed in order to run the ruby command. Rubypick by itself isn't sufficient and it requires rubygems to run (not sure why on that part).
